src/index.js

```diff
@@ -1,24 +1,22 @@
 // @flow
 
-import type { PackageAtVersion } from './types';
+import type { PackageAtVersion, PacoteOptions } from './types';
 
 const getAllDependencies = require('./get-dependencies');
 const getLatestDependencies = require('./get-latest-dependencies');
 const { resolveDependencies, formatGraphWithoutCircularDeps } = require('./format-dependencies');
 const _ = require('lodash');
 
-function buildGraph(rootPackage: PackageAtVersion, pacoteOptions: Object) {
+function buildGraph(rootPackage: PackageAtVersion, pacoteOptions: ?PacoteOptions) {
   return getAllDependencies(rootPackage, pacoteOptions).then(result => (
-    getLatestDependencies(
-      _.uniq(result.dependencies.map(npmPackage => npmPackage.name)),
-    )
-    .then(latestVersions => (
-      formatGraphWithoutCircularDeps(
-        { name: rootPackage.name, version: result.resolvedDependencies[`${rootPackage.name}@${rootPackage.version}`] },
-        resolveDependencies(result),
-        latestVersions,
-      )
-    ))
+    getLatestDependencies(_.uniq(result.dependencies.map(npmPackage => npmPackage.name)))
+      .then(latestVersions => (
+        formatGraphWithoutCircularDeps(
+          { name: rootPackage.name, version: result.resolvedDependencies[`${rootPackage.name}@${rootPackage.version}`] },
+          resolveDependencies(result),
+          latestVersions,
+        )
+      ))
```

Comment:
Again is this prettier? I actually don't really like most of the formatting suggestions it's made - I think the original is much easier to read.